scripts/clientSeed.js

```diff
@@ -0,0 +1,49 @@
+// ---------------- TEST DATA   ----------------
+// a page to input fake data in JSON FORMAT
+
+var mongoose = require("mongoose");
+const db = require("../models");
+
+mongoose.connect(
+  process.env.MONGODB_URI || "mongodb://localhost/neighborfavors_db"
+);
+
+const clientSeed = [
+  {
+    firstName: "Siberyan",
+    lastName: "Husky",
+    email: "shusky@gmail.com",
+    password: "Wh!teClaw7",
+    neighborhood: "Sunset",
+    bio: "A regular human bean",
+    photoUrl: "https://tinypic.com/f",
+    residenceType: "house",
+    security: "key",
+    attendant: false,
+    date: Date()
+  },
+  {
+    firstName: "Drew",
+    lastName: "Hermit",
+    email: "dhermit@gmail.com",
+    password: "Pa$$word",
```

Comment:
When a uses signs up the password doesnt get saved to the database. its actually a long encrypted string that needs to be decrypted when they sign in. How can you mock this behavior